wdl/wdlParser.py

```diff
@@ -231,6 +232,53 @@ def module():
 
     return mod_type
 
+
+# -----------------------------------------------------------------------------
+# @fn     lvds
+# @brief  rules for the LVDSL keyword
+# @param  string slotNumber
+# @param  int type
+# @return none, appends to global variable "lvdsOutput"
+# -----------------------------------------------------------------------------
+def lvds(slot_number):
+    """
+    These are the rules for the LVDSL keyword, encountered while parsing
+    the SLOT command for the modules (.mod) file. Required format is
+    LVDSL # "label";
+
+    where # is any number
+    where "label" is a string
+    """
+    global token
+    global lvdsOutput
```

Comment:
I know it's how the rest of the code is, but is there a reason we actually need `global` here rather than `nonlocal`? 

The latter would in principle allow an easier refactor later